src/bindings/lua/flux-lua.c

```diff
@@ -650,10 +656,12 @@ static int l_f_zi_resp_cb (lua_State *L,
 }
 
 static int create_and_push_zmsg_info (lua_State *L,
-        flux_t *f, int typemask, flux_msg_t **msg)
+                                      flux_t *f,
+                                      int typemask,
+                                      flux_msg_t **msg)
 {
-    struct zmsg_info * zi = zmsg_info_create (msg, typemask);
-    zmsg_info_register_resp_cb (zi, (zi_resp_f) l_f_zi_resp_cb, (void *) f);
+    struct zmsg_info *zi = zmsg_info_create (msg, typemask);
+    zmsg_info_register_resp_cb (zi, (zi_resp_f)l_f_zi_resp_cb, (void *)f);
```

Comment:
I think we will want to add `SpaceAfterCStyleCast : true`

Reply:
Added in https://github.com/flux-framework/flux-core/pull/2161/commits/b685717b59912bcb98f4a1d21676272d9224511b

That commit can be dropped if others disagree with the change.

Reply:
I always have written casts so they kiss up to the variable they apply to, so that feels a bit strange.   I could easily be talked out of that though.

Reply:
I slightly lean towards @garlick's preference, though I don't care that much.

If anything, if we disable `SpaceAfterCstyleCast` there will be less changes in this PR.

Reply:
Dropped that commit in the latest push